src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,61 @@
+import { useEffect, useState } from 'react';
+import { Person } from '../../types';
+import { getPeople } from '../../api';
+import { Loader } from '../Loader';
+import { PeopleTable } from '../PeopleTable/PeopleTable';
+
+export const PeoplePages = () => {
+  const [dataPersons, setDataPersons] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [loadError, setLoadError] = useState(false);
+
+  const sortPersons = (persons: Person[]) => {
+    return persons.map(person => ({
+      ...person,
+      mother: persons.find(mom => mom.name === person.motherName),
+      father: persons.find(dad => dad.name === person.fatherName),
+    }));
+  };
```

Comment:
Why function named `sort`?
Move it to separate file/function out from component